consensus/src/messages/mod.rs

```diff
@@ -257,6 +258,11 @@ impl RequestCommon for RequestBlock {
     type Response = Result<Block, BlockError>;
     const MAX_REQUESTS: u32 = MAX_REQUEST_RESPONSE_BLOCK;
 }
+test_max_req_size!(
```

Comment:
I'd move this to the end of the file.

Reply:
We discussed this briefly and in the end kept it at the current position.